Some small improvements to the Mushrooms example (#5982)
chainer/chainer: docs/source/glance.rst

   :width: 835px
   :height: 353px

Now that we have the training :class:`iterator <chainer.dataset.Iterator>` and :class:`optimizer <chainer.Optimizer>` set up, we link them both together into the :class:`updater <chainer.training.Updater>`. The :class:`updater <chainer.training.Updater>` uses the minibatches from the :class:`iterator <chainer.dataset.Iterator>`, does the forward and backward processing of the model, and updates the parameters of the model according to the :class:`optimizer <chainer.Optimizer>`. Setting the ``device=-1`` sets the device as the CPU. To use a GPU, set ``device`` equal to the number of the GPU, usually ``device=0``.

.. literalinclude:: ../../examples/glance/glance.py
   :language: python
   :linenos:
   :lines: 49-50
   :lineno-start: 49

> Finally we create a :class:`Trainer <chainer.training.Trainer>` object. The ``trainer`` processes minibatches using the ``updater`` defined above until a certain stop condition is met, and allows to use extensions at certain phases of the training. We set it up to run for 50 epochs and store all files created by extensions (see below) in the ``result`` directory.

Reviewer: ```suggestion
Finally we create a :class:`Trainer <chainer.training.Trainer>` object. The ``trainer`` processes minibatches using the ``updater`` defined above until a certain stop condition is met and allows the use of extensions during the training. We set it to run for 50 epochs and store all files created by the extensions (see below) in the ``result`` directory.
```

   :width: 835px
   :height: 353px

> Extensions can be used to execute code at certain events during the training, for example every epoch or every 1000 iterations. This mechanism is used in Chainer to evaluate models during training, print progress messages, or dump intermediate model files.

Reviewer: ```suggestion
Extensions can be used to execute code at certain events during the training, such as every epoch or every 1000 iterations. This mechanism is used in Chainer to evaluate models during training, print progress messages, or dump intermediate model files.
```
